### DISCUSSION TITLE
WSL2 cuda support on Win10 20H2 update?

### DISCUSSION THREAD

**Discussion by kroxiksut** [microsoft/WSL#6265] [General] [answered]
Does anyone have information when you can expect support for Cuda WSL in stable versions of Windows?

**Comment by ghost** [comment, marked as answer]
It will be supported in an upcoming version of Windows. See #6148

**Comment by WSLUser** [comment]
The 2nd release in the year is focused primarily on bug fixes after all the fiascos regarding upgrades. So no new major features until spring time release, this includes cuda support.
